geomstats/geometry/spd_matrices.py

```diff
@@ -861,6 +879,24 @@ def squared_dist(self, point_a, point_b, **kwargs):
 
         return trace_a + trace_b - 2 * trace_prod
 
+    def injectivity_radius(self, base_point):
+        """Compute the upper bound of the injectivity domain.
+
+        This is the smallest eigen value of the base point.
+
+        Parameters
+        ----------
+        base_point : array-like, shape=[..., n, n]
+            Point on the manifold.
+
+        Returns
+        -------
+        radius : float
+            Injectivity radius.
+        """
+        eigen_values = gs.linalg.eigvalsh(base_point)
+        return eigen_values[..., 0]
```

Comment:
Sorry, I probably told you something wrong: it is the square-root of the smallest eigenvalue. (It is the smallest eigenvalue for the Euclidean metric.)